galaxy.yml

```diff
@@ -0,0 +1,14 @@
+namespace: "hpe"
```

Comment:
as a standard practice .. galaxy.xml file should always be present under the collection folder. in this it will be "nimble".
let's not move it to repo folder.a user can always give the below command to install the collection properly

`ansible-galaxy collection install git+https://github.com/hpe-storage/nimble-ansible-modules.git#ansible_collection/hpe/nimble,rel-1.0.0`

we can modify the README file with this command and leave the galaxy.xml in the same folder.

- each collection will have its own galaxy.xml. and in future let's say we develop another collection in same repo then it will be a problem

Reply:
Thanks for clarifying. Fixed.